examples/decoding/plot_decoding_spatio_temporal_source.py

```diff
@@ -44,7 +49,7 @@
 
 # Setup for reading the raw data
 raw = io.read_raw_fif(raw_fname, preload=True)
-raw.filter(2, None, fir_design='firwin')  # replace baselining with high-pass
+raw.filter(0.1, None, fir_design='firwin')
```

Comment:
So 0.1 highpass but still `baseline=None`. Is this recommended in general, or case-by-case it makes a difference (and MGH data are very clean so it's not necessary?)